e2e/ChangeDisplayNameTest.e2e.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import { screen } from '../react/features/conference/components/native/routes';
+
+describe('Jitsi Meet App', () => {
```

Comment:
Let's give each test a more descriptive name?

Reply:
Moved everything to WelcomePageTest.e2e.js.